source/_integrations/google_weather.markdown

```diff
@@ -0,0 +1,50 @@
+---
+title: Google Weather
+description: Instructions on how to integrate Google Weather within Home Assistant.
+ha_category:
+  - Weather
+ha_release: '2025.7'
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_domain: google_weather
+ha_codeowners:
+  - '@tronikos'
+ha_integration_type: service
+ha_quality_scale: bronze
+ha_platforms:
+  - weather
+---
+
+This {% term integration %} uses [Google Weather](https://developers.google.com/maps/documentation/weather) as a source for weather data for your location.
+
+## Prerequisites
+
+You need an API key. Follow instructions [here](https://developers.google.com/maps/documentation/weather/get-api-key) for creating an API key.
+
+The Weather API is in the pre-GA Preview stage, and there is no charge for API usage during Preview.
+For pricing once it's in GA see [here](https://developers.google.com/maps/documentation/weather/usage-and-billing).
+It will be free as long as you stay under 10,000 requests per month.

```

Comment:
You can put this in an informational note. :)

```suggestion
{% note %}
The Weather API is in the pre-GA Preview stage, and there is no charge for API usage during Preview.
For pricing once it's in GA see [here](https://developers.google.com/maps/documentation/weather/usage-and-billing).
It will be free as long as you stay under 10,000 requests per month.
{% endnote %}
```